### DISCUSSION TITLE
Vitocal 250-A Access Point vs. Cloud Connection

### DISCUSSION THREAD

**Discussion by Philip-Wiege** [open3e/open3e#155] [Q&A]
Hello everyone,

I just noticed that I can enable the access point on my Vitocal 250-A IDU and still keep it connected to the ViCare cloud.
Open3E works fine over DoIP in this parallel operation. Was this always possible or is this a new feature from a SW update?

Hi zusammen,
ich habe gerade bemerkt, dass ich an meiner Vitocal 250-A IDU den Access Point einschalten kann und die Anlage trotzdem mit der ViCare Cloud verbunden bleibt. OPen3E funktioniert über DoIP ganz normal im Parallelbetrieb zur Cloud. Ging das schon immer oder kam das erst nach einem SW Update?

**Comment by flecke-m** [comment]
Hm bei meiner Vitocal 222-s geht das leider nicht.

Sobald ich den AP anschalte wir die Verbindung zur VI Cloud gekappt. Auch wenn die Verbindung zum Access Point bestehen bleibt...

**Reply by Philip-Wiege** [reply]
Schaltest du den Access Point über das Service-Menü oder über das "normale" Menü ein? Welche SW Version hast du auf deinem Gerät?

Grüße
Philip

**Reply by flecke-m** [reply]
Ich habe es erst mit dem normalen probiert und dann im Service Menü, verhalten ist gleich. Software Stand ist noch 2323 (trotz Auto Update...).

**Reply by Philip-Wiege** [reply]
Ah ok. Meine ist bei der Inbetriebnahme durch den Viessmann Techniker auf 2404 geupdated worden. Auch wenn in den Release Notes nichts dazu steht, gehe ich davon aus, dass es daran liegt.

Danke für das Feedback.

**Reply by flecke-m** [reply]
Ja mal schauen, ich hoffe ich bekomme das Update irgendwann, wäre für mich super Klasse mein Raspberry hängt 1,5m von der WP entfernt und meine Geräte sind nicht über Can Bis verbunden.

**Reply by Philip-Wiege** [reply]
Was meinst du mit nicht über CAN Bus verbunden? Meinst du nur, dass dein Raspberry Pi noch nicht am CAN Bus hängt? Die 222-S ist doch normales OneBase Gerät und sollte CAN Schnittstellen haben?

**Reply by flecke-m** [reply]
Ja ich habe mich schlecht ausgedrückt ;)

Ich habe die Vitocal, ne VX3 und eine Vitoair FS. Alle drei sind trotz Viessmann Inbetriebnahme "Standalone" und nicht über Can verbunden. Aktuell habe ich die VX3 mit open3e und e3oncan laufen (wegen dem E380), damit ich die PV Daten gut schön habe (da ich mich ein SMA Wechselrichter habe).

Tja um die FS300 mit der Vitocal über Can zu verbinden muss die FS300 nochmal neu Inbetrieb genommen werden und das ohne Viguide Account traue ich mich nicht (wenn es überhaupt geht). Daher muss ich noch die zwei Anlagen anschließen und plane noch zwei USB CAN Adapter, da wäre der AP Modus prima um es einfacher zu haben.

**Reply by 0bscur3** [reply]
Ich habe es noch nicht ausprobiert, aber um Probleme bei Garantieansprüchen zu vermeiden, wäre es wahrscheinlich besser, die Verbindung über DOIP anstatt CAN aufzubauen. Daher würde ich bei funktionierender DOIP mit Cloud gerne darauf umstellen.

> Ah ok. Meine ist bei der Inbetriebnahme durch den Viessmann Techniker auf 2404 geupdated worden. Auch wenn in den Release Notes nichts dazu steht, gehe ich davon aus, dass es daran liegt.

Meine Vitocal 252A ist auch noch auf 2323. @Philip-Wiege Wo finde ich denn die Release Notes von denen du sprichst? Bisher wurde ich nicht fündig und laut Viessmann-Community werden die auch nicht öffentlich geteilt.

**Reply by Philip-Wiege** [reply]
Für die SW2440 gibt es Release Notes in ViBooks. Irgendwie hat meine Anlage doch keine Backend Verbindung mehr, wenn ich den Access Point einschalte. Das wahr wohl leider ein Fehlalarm von mir...
